Tests/ReactorKitTests/ReactorTests.swift

```diff
@@ -358,8 +358,59 @@ final class ReactorTests: XCTestCase {
     XCTAssertNil(weakAction)
     XCTAssertNil(weakState)
   }
+
+  func testState() {
+    // given
+    let reactor = TestCounterReactor()
+    let disposeBag = DisposeBag()
+    var receivedValues: [Int] = []
+    var receivedLoadings: [Bool] = []
+
+    reactor.state(\.value)
+      .distinctUntilChanged()
+      .subscribe(onNext: { value in
+        print(value)
```

Comment:
could you delete some codes for debugging?
```suggestion
```

Reply:
Oops! I missed it.